### PR TITLE
Add Spotify and WhatsApp data

### PR DESCRIPTION
**Pull request by marcpems**
adding WhatsApp, Spotify items, corresponding new categories and icons

### CODE REVIEW

File: src/content/categories/audio.md
Comment (anthony-linaro) [inline src/content/categories/audio.md:1]: This does not match with the category name given under spotify - it needs to either be "audio" under spotify, or this file need to be renamed to "music"

File: src/content/categories/app.md
Comment (anthony-linaro) [inline src/content/categories/app.md:1]: I think a better description would work here - are me meaning MS store apps here?

File: src/content/projects/spotify.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+name: Spotify
+icon: spotify.svg
+categories: [Music, App]
```
Comment (anthony-linaro) [inline src/content/projects/spotify.md:4]: See above comment

**Review by anthony-linaro — CHANGES_REQUESTED** [review]
A quick initial pass

File: src/content/projects/spotify.md (hunk 1 above)
Comment (marcpems) [reply src/content/projects/spotify.md]: thx. I'll resolve these and update

File: src/content/categories/msstore.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+---
+name: msstore
```
Comment (anthony-linaro) [inline src/content/categories/msstore.md:2]: This is supposed to be a human-readable label - "Microsoft Store" perhaps?

File: src/content/projects/spotify.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+name: Spotify
+icon: spotify.svg
+categories: [Audio, App, MSStore]
```
Comment (anthony-linaro) [inline src/content/projects/spotify.md:4]: These need to be lower case

File: src/content/projects/whatsapp.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+---
+name: WhatsApp
+icon: WhatsApp.png
+categories: [Comms, App, MSStore]
```
Comment (anthony-linaro) [inline src/content/projects/whatsapp.md:4]: These need to be lower case

**Review by anthony-linaro — CHANGES_REQUESTED** [review]
A few more changes to be made.

Also, please can you confirm the licensing for where you have sourced the logos from please?

**Review by pcolmer — APPROVED** [review]

**Review by pcolmer — APPROVED** [review]

**Pull request merged**